PyRDF/CallableGenerator.py

```diff
@@ -103,7 +103,8 @@ def mapper(node_cpp, node_py=None, rdf_range=None):
                                                              start, end)
                     # Create a partial snapshot on the current range
                     pyroot_node = RDFOperation(operation.args[0],
-                                               path_with_range)
+                                               path_with_range,
+                                               *operation.args[2:])
```

Comment:
I would like to have only one assignment to `pyroot_node` this if condition should serve only to change the path of the distributed file, as in 
```
if rdf_range and operation.name == "Snapshot":
    # change the path
    operation.args[1] = path_with_range

# leave pyroot_node assignment outside of if statement
pyroot_node = RDFOperation(*operation.args, **operation.kwargs)
```

the only problem with that is that `operation.args` is a tuple so it is immutable. This could be solved by casting it to a list, either in this file or even better in the `Operation.py` class file